<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentFrequencyWizardPage</class>
 <widget class="QWizardPage" name="PaymentFrequencyWizardPage">
  <attribute name="title">
    <string>Payment frequency</string>
  </attribute>
  <layout class="QVBoxLayout">
    <property name="margin">
      <number>11</number>
    </property>
    <property name="spacing">
      <number>6</number>
    </property>
    <item>
      <widget class="QLabel" name="TextLabel11_2">
        <property name="text">
          <string>
How often will there be payments made to this loan?</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer54">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>49</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <spacer name="Spacer57">
            <property name="sizeHint">
              <size>
                <width>39</width>
                <height>0</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="KMyMoneyFrequencyCombo" name="m_paymentFrequencyUnitEdit"/>
        </item>
        <item>
          <spacer name="Spacer56">
            <property name="sizeHint">
              <size>
                <width>56</width>
                <height>0</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer55">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>49</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
 <customwidget>
  <class>KMyMoneyFrequencyCombo</class>
  <extends>KComboBox</extends>
  <header>kmymoneyfrequencycombo.h</header>
 </customwidget>
 <customwidget>
  <class>KComboBox</class>
  <extends>QComboBox</extends>
  <header>kcombobox.h</header>
 </customwidget>

 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
